### PR TITLE
FIX: csp numpy 1.12 max takes an array

### PR DESCRIPTION
**Pull request by kingjr**
closes 3578

### CODE REVIEW

File: mne/decoding/csp.py [hunk 1]
```diff
@@ -601,7 +601,7 @@ def _ajd_pham(X, eps=1e-6, max_iter=15):
 
                 tmp = np.sqrt(omega21 / omega12)
                 tmp1 = (tmp * g12 + g21) / (omega + 1)
-                tmp2 = (tmp * g12 - g21) / np.max(omega - 1, 1e-9)
+                tmp2 = (tmp * g12 - g21) / np.maximum(omega - 1, 1e-9)
```
Comment (agramfort) [inline mne/decoding/csp.py]: omega is a float so you even be 

max(omega - 1, 1e-9)

no numpy

**Comment by Eric89GXL** [comment]
I think this is actually a bugfix, related to 1.12 only because they raise an error instead of warning. Is omega a vector or scalar? If it's vector it's actually a different operation in all cases. If a scalar, different only if `omega - 1 < 1e-9`.

**Comment by Eric89GXL** [comment]
Never mind @agramfort said it's a float (scalar) already

**Comment by kingjr** [comment]
yes it's a float, do I close?

**Comment by Eric89GXL** [comment]
No, it's still a bug. @agramfort is just saying to use `max` instead of `np.max`. Look at the arguments of `np.max`, the second is the *axis* not the value to compare with. So in `master`, any value < 1e-9 (like zero!) will not be scaled up to 1e-9 like it's designed to do.

**Comment by kingjr** [comment]
Then it's fine with np.maximum; but it'll change it to max for simplicity

**Comment by agramfort** [comment]
lgtm

@kingjr merge when CIs are happy

now you can press the green button :)

**Comment by kingjr** [comment]
> now you can press the green button :)

I'm so excited ;)

**Comment by kingjr** [comment]
My first merge to master is probably the smallest commit of the package. Nice.

**Comment by jona-sassenhagen** [comment]
Nope; https://github.com/mne-tools/mne-python/pull/3469 :)

**Comment by agramfort** [comment]
congrats :)

**Pull request merged**